test/.stylelintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["@netcentric/stylelint-config"],
```

Comment:
Can we use some custom file instead of installing a plugin (which also have 2 additional dependencies)

```
{
  "extends": ["./myExtendableConfig"],
  "rules": {
    "alpha-value-notation": "number"
  }
}
```
`myExtendableConfig.js`

```
eg: https://github.com/stylelint/stylelint-config-recommended/blob/main/index.js
```

Reply:
But this devDependency is only for us, users of our fe-build will actually never install it.
Then probably all is fine.

Reply:
The idea behind is to have a test project with the same settings we enforce at NC, like using @netcentric/stylelint-config. And as you said, it is a devDependency so it won't affect end users.